doc/source/parameters/dem/insertion_info.rst

```diff
@@ -51,6 +51,11 @@ In this subsection, insertion methods which are ``volume``, ``plane``, ``list``
 
     # If method = file
     set insertion file name                            = particles.input
+
+    # If method = clear_and_add
+    set list of input files                            = particles.input
+    set clearing box points coordinates                = 0., 0., 0.: 1., 1., 1.
```

Comment:
This falls in the problem of not being suitable for spherical and cylindrical domains. Could you do a mesh instead of a box? Then it could be a deal.ii mesh or a custom mesh.

Reply:
I like the idea, but currently I don't know how to do that and it won't be useful for my problem. I prefer not to put to much time into something that will probably never be used.

I think this would be a good addition long term, especially if this addition is seamless with the current implementation.

Reply:
Also, we could do the same thing for the volume insertion.

Reply:
This aspect is orthogonal. Here the idea is that you have a clearing/insertion mechanism. The fact that we don't have another box than "square box" is an another issue that we can address in another PR. One PR, one feature.